firmware/brake/kia_soul_ev_niro/include/status.h

```diff
@@ -0,0 +1,79 @@
+/**
+ * @file status.h
+ * @brief Control of the status (LED) system.
+ *
+ */
+
+
+#ifndef _OSCC_STATUS_H_
+#define _OSCC_STATUSL_H_
+
+
+#include <stdint.h>
+
+
+
+// ****************************************************************************
+// Function:    init_status
+//
+// Purpose:     Configured status LED pins as output and turns them on.
+//
+// Returns:     void
+//
+// Parameters:  void
+//
+// ****************************************************************************
+void init_status( void );
+
+// ****************************************************************************
+// Function:    status_ledsOff
+//
+// Purpose:     Turns off both status LEDs
+//
+// Returns:     void
+//
+// Parameters:  void
+//
+// ****************************************************************************
+void status_ledsOff( void);
```

Comment:
Most of the code that I've seen uses `snake_case` but this function name mixes camel case and snake case, is there any reason for this?